### DISCUSSION TITLE
Using salinity to detect water level.

### DISCUSSION THREAD

**Discussion by LTrainExpress** [tagyoureit/nodejs-poolController#768] [Ideas]
Has anyone tried or considered using water salinity to roughly detect water level? I am hoping to measure salinity of the pool then add 2 inches of water or 5cm and remeasure the salinity and create a custom coefficient for how much the salinity drops per inch of water. Then every time I add salt all I should have to do is measure the current salinity and water level from the top of the tile at a consistent location. From here I should get notified when the water is too high or too low based on the salinity. When I drain the pool I will have to enter in the new water level and and maybe current salinity like adding salt, but I am hoping that the coefficient of salinity delta per inch of water should never change.  Any thoughts or comments?

**Comment by rstrouse** [comment]
Unfortunately, all salinity measurements are based on total dissolved solids.  You will find this only to be accurate within +-500ppm. 
 Probably not close enough even for government work.  The second your TDS changes then whammo your coefficient will change.
